backend/src/api/endpoints/user.py

```diff
@@ -88,73 +93,26 @@ async def delete_by_id(
     response_model=List[ProjectRead],
     summary="Returns all Projects of the User",
     description="Returns all Projects of the User with the given ID",
+    # Only users themselves can see what projects they are in
+    dependencies=[is_authorized(ActionType.READ, crud_user, "user_id")],
 )
 async def get_user_projects(
     *, user_id: int, db: Session = Depends(get_db_session)
 ) -> List[ProjectRead]:
-    # TODO Flo: only if the user has access?
     db_obj = crud_user.read(db=db, id=user_id)
     return [ProjectRead.from_orm(proj) for proj in db_obj.projects]
 
 
-@router.get(
```

Comment:
Why did you remove these routes? Don't we need them?

Reply:
We are not using them, and I talked to Tim, he said won't need them in the future either.